### PR TITLE
remove encoding decoding

### PR DESCRIPTION
**Pull request by dragos-rebegea**

### CODE REVIEW

File: totp.go [hunk 1]
```diff
@@ -454,65 +451,17 @@ func (otp *Totp) ToBytes() ([]byte, error) {
 		}
 	}
 
-	// encrypt the TOTP bytes
-	engine, err := cryptoengine.InitCryptoEngine(otp.issuer)
```
Comment (ssd04) [inline totp.go:458]: remove also comment line regarding encryption? L340

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module github.com/multiversx/twofactor
+
+go 1.17
```
Comment (ssd04) [inline go.mod]: in multi-factor-auth repo, for example, we changed to 1.18, maybe change here also

**Review by sstanculeanu — COMMENTED** [review]
only Darius' comments

File: totp.go (hunk 1 above)
Comment (dragos-rebegea) [reply totp.go:458]: removed

**Review by sstanculeanu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (dragos-rebegea) [reply go.mod]: updated

**Pull request merged**